pkg/model/crd.go

```diff
@@ -387,6 +388,81 @@ func (r *CRD) GetOutputWrapperFieldPath(
 	return &opConfig.OutputWrapperFieldPath
 }
 
+// GetOutputShape returns the Output shape for given operation.
+func (r *CRD) GetOutputShape(
+	// The operation to look for the Output shape
+	op *awssdkmodel.Operation,
+) (*awssdkmodel.Shape, error) {
+	if op == nil {
+		return nil, nil
+	}
```

Comment:
We don't need this block - this means that even if `err` is nil we should perform a "nillity" test on the shape as well. We just have to make sure that either we return and nil `error` or a nil `shape` but never both at the same in the logic bellow.

Reply:
Thanks for the comment, I have updated the code.